index.js

```diff
@@ -23,7 +22,7 @@ function load() {
 
     // The JSON data is independent of the actual file
     // hierarchy, so it is essential to extend "deeply".
-    result = extend(true, result, extra);
+    result = extend(result, extra);
```

Comment:
```suggestion
    extend(result, extra);
```

Might as well ignore the return value, since `extend` is now more like `Object.assign`.